src/Images/SampleDistanceSlider.jsx

```diff
@@ -1,66 +1,73 @@
 import React, { useEffect, useRef } from 'react'
-import { useActor } from '@xstate/react'
-import { Icon, Slider, Tooltip } from '@mui/material'
+import { useSelector } from '@xstate/react'
 import { sampleDistanceIconDataUri } from 'itk-viewer-icons'
 import applyContrastSensitiveStyleToElement from '../applyContrastSensitiveStyleToElement'
 import '../style.css'
+import Button from 'react-bootstrap/Button'
+import Form from 'react-bootstrap/Form'
+import Image from 'react-bootstrap/Image'
+import Tooltip from 'react-bootstrap/Tooltip'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import cn from 'classnames'
 
-function SampleDistanceSlider(props) {
+const SampleDistanceSlider = React.memo(function SampleDistanceSlider(props) {
   const { service } = props
```

Comment:
We should not need `React.memo` if the only prop we are passing is `service` which should not change over the lifetime of the app.  After we get these PRs merged, maybe we look up the component tree and tackle the useActors at the top levels and stop the top components from re-rendering on each state change.  If they stop rendering all the time, then we probably don't need React.memo on all of our little components.

Reply:
Got it! So it makes sense to not worry about the re-renderings now and tackle that in the next step.